### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by tsmithv11**
Added additional details and screenshots.

## Description

Added more detail to the description, including how to add a relevant rule as well as some screenshots.

## Motivation and Context

Screenshots make it easier to digest and understand the description.

## How Has This Been Tested?

Tested locally and in GitHub.

## Checklist

- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,16 +33,34 @@ jobs:
...
 
-## Inputs
+
+## Properties
+
+### Inputs
 | Input | Description | Required? | Default |
 |---|---|---|---|
-| `pcc_console_url` | URL of your Prisma Cloud Compute Console | Yes |  |
+| `pcc_console_url` | URL of your Prisma Cloud Console | Yes |  |
```
Comment (wfg) [inline README.md:42]: Hey @tsmithv11, I think the distinction is important in this case. Dropping Compute here would likely confuse things as we have two URLs with SaaS: one for Prisma Cloud (former RedLock) APIs and Prisma Cloud Compute (former Twistlock) APIs. "URL of your Prisma Cloud Console" tells me `app3.prismacloud.io`, not `us-west1.cloud.twistlock.com/us-3-12345`.

**Review by wfg — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # Prisma Cloud Scan Action
-Scan container images for vulnerabilities and compliance issues using this GitHub action.
-This action is a wrapper around [twistcli](https://docs.twistlock.com/docs/compute_edition/tools/twistcli_scan_images.html) which connects to the specified Prisma Cloud Compute Console for vulnerability and compliance policy and metadata.
+The Prisma Cloud Scan Action will scan container images for vulnerabilities and compliance issues using this GitHub action. Receive immediate feedback about image vulnerabilities and compliance violations both in GitHub and in the Prisma Cloud Console. Block merges that do not meet your compliance requirements, such as updates with critical vulnerabilities.
```
Comment (wfg) [inline README.md:2]: @tsmithv11 "using this GitHub action" at the end of the first sentence seems redundant to me.

Comment (tsmithv11) [reply README.md]: Good catch

File: README.md (hunk 1 above)
Comment (tsmithv11) [reply README.md]: That makes sense. Updated

**Comment by wfg** [comment]
Looks great! Thanks @tsmithv11

**Pull request merged**